src/main/java/org/jboss/arquillian/extension/jacoco/client/ApplicationArchiveInstrumenter.java

```diff
@@ -47,6 +47,10 @@
    @Inject
    private Instance<JacocoConfiguration> config;
 
+   public void setConfig(Instance<JacocoConfiguration> config) {
+      this.config = config;
+   }
```

Comment:
@bartoszmajsak We can't access injected variable using reflection to set default value as it's private . I think let's stick with setConfig.